updated press page content

danielfmiranda opened the pull request:
# Description

<!-- Describe the PR here -->

Link to sample test page: https://foundation-s-feature-11-nobyi2.herokuapp.com/en/privacynotincluded/about/press/
Related PRs/issues: #11138



This PR updates the [PNI press page](https://foundation.mozilla.org/en/privacynotincluded/about/press/) with the updated copy found [here](https://docs.google.com/document/d/1InL0XlA1XuJsu-MoSkRk7acX-Ni-j0sb52OqTQ-mBt4/edit#heading=h.sa73c5locnym).

Zoe-PNI commented:
thank you, Daniel! 

- could you please remove the word "Recognition" (first image)

- and add add hyperlink source for the second NPR quote in the same style as the first? (second image)
here is the link https://www.npr.org/2019/12/02/784063267/mozillas-latest-privacy-not-included-buyers-guide 


<img width="827" alt="Screenshot 2024-01-25 at 10 05 41 AM" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/145496977/37f5c1b2-9a3c-47b5-80e6-0a25725b6a15">
<img width="1047" alt="Screenshot 2024-01-25 at 10 06 19 AM" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/145496977/b1b724fd-c312-4832-8422-ed0abe05d4e5">

mtdenton commented:
Hi @Zoe-PNI ! Is the copy document final after the changes you requested above? If this is something that Daniel missed, please disregard.

@danielfmiranda let's hold on further commits until we have confirmation from Zoe that this is final state of the copy doc.

Thanks both!

Zoe-PNI commented:
@mtdenton yes! that's correct. thank you! cc: @danielfmiranda

danielfmiranda commented:
Thanks @mtdenton @Zoe-PNI!

@Zoe-PNI, I have removed "Recognition" from the copy, and also updated the second quote to link to the provided NPR article as requested.

Can you please take another look whenever you find a free moment and let me know if everything looks OK? 

Thanks again!

Zoe-PNI commented:
Good to go on my side, thank you @danielfmiranda !

jhonatan-lopes commented on network-api/networkapi/templates/pages/buyersguide/about/press.html:
```diff
@@ -10,200 +10,222 @@ <h1 class="tw-h2-heading">
-            {{ name}} at <a href="mailto:{{ email }}">{{email }}</a>. Our researchers can answer questions about
-            specific products, or speak to broader trends in the realm of privacy, security, and trustworthy AI.
-            We can also provide high-resolution graphics and gifs from our guide.
+        {% blocktrans with best_of_url="/privacynotincluded/categories/best-of/" trimmed %}
```
I don't like that this is a hardcoded URL but I guess it works for this page :grimacing:

mmmavis replied:
I have the same concern

jhonatan-lopes commented on network-api/networkapi/templates/pages/buyersguide/about/press.html:
```diff
@@ -10,200 +10,222 @@ <h1 class="tw-h2-heading">
+                {% endblocktrans %}
+            </h3>
+            <div class="attribution-with-link tw-inline-flex tw-gap-2">
+                {% blocktrans with url="https://www.npr.org/2023/09/16/1200012617/experts-say-modern-cars-can-violate-consumer-privacy" trimmed %}
+                    <p class="tw-h6-heading ">
+                        — Scott Detrow,
+                    </p>
+                    <a class="tw-blue-80 tw-text-xs" href="{{ url }}">
+                        NPR
+                    </a>
+                {% endblocktrans %}
```
This one doesn't seem like it needs a `blocktrans` unless we need to localise "NPR". In that case, I would put just a translation tag around NPR instead of a `blocktrans` for the whole block

danielfmiranda replied:
Great catch @jhonatan-lopes! I have updated this in my latest commit.

jhonatan-lopes commented on network-api/networkapi/templates/pages/buyersguide/about/press.html:
```diff
@@ -10,200 +10,222 @@ <h1 class="tw-h2-heading">
-
+            </h3>
+            <div class="attribution-with-link tw-inline-flex tw-gap-2">
+                {% blocktrans with url="https://www.npr.org/2019/12/02/784063267/mozillas-latest-privacy-not-included-buyers-guide" trimmed %}
+                    <p class="tw-h6-heading ">
+                        — Steve Inskeep,
+                    </p>
+                    <a class="tw-blue-80 tw-text-xs" href="{{ url }}">
+                        NPR
+                    </a>
+                {% endblocktrans %}
```
Same thing as up there

danielfmiranda replied:
Also updated!

mmmavis reviewed (changes requested):
PR looks good. Just one flag that we need to remove `frameborder` from `iframe` as that attribute is obsolete on iframe. We should be able to achieve the same result with Tailwind (whatever the border 0 class is).

danielfmiranda commented:
Thank you both for taking a look! @mmmavis @jhonatan-lopes 

I have updated this PR with the following changes, based on your feedback:

- I have removed the `frameborder` attribute from the youtube-provided iframe. No additional classes or changes were needed! 
- I have removed `{% transblock %}` from both NPR quote attributions as the tag was not needed.
- Regarding the hardcoded URL, after some discussion with Jhonatan, I have updated the template to grab the appropriate URL using `{% routablepageurl page 'category-view' slug='best-of' as best_of_url %}`. Technically there is still some hardcoding of the slug using this method, but being that this is a one-off page template, and that the users requested this page be updated before the valentines day PNI push, I believe it should do for now.

mmmavis reviewed (approved):
Awesome work! Thanks @danielfmiranda !

jhonatan-lopes reviewed (approved):
LGTM
